ticketutil/servicenow.py

```diff
@@ -41,11 +53,19 @@ def __init__(self, url, project, auth=None, ticket_id=None):
 
         self.s = self._create_requests_session()
         if ticket_id:
-            self.ticket_id = ticket_id
-            self.ticket_content = self.get_ticket_content()
-            self.sys_id = self.ticket_content['sys_id']
-            self.ticket_rest_url = self.rest_url + '/' + self.sys_id
-            self.ticket_url = self._generate_ticket_url()
+            self.set_ticket_id(ticket_id)
+
+    def set_ticket_id(self, ticket_id):
+        """
+        Sets ticket vars for the current ticket object.
+        :param ticket_id: Ticket id you would like to set.
+        :return:
+        """
+        self.ticket_id = ticket_id
+        self.ticket_content = self.get_ticket_content()
+        self.sys_id = self.ticket_content['sys_id']
+        self.ticket_rest_url = self.rest_url + '/' + self.sys_id
+        self.ticket_url = self._generate_ticket_url()
 
     def get_ticket_content(self, ticket_id=None):
         """
```

Comment:
Line 74 below should say ticket_id and ticket number instead of ticked_id and ticked number